pkg/broker/broker.go

```diff
@@ -789,14 +769,41 @@ func (a AnsibleBroker) Unbind(
 		return nil, errors.New(errMsg)
 	}
 
+	params := make(apb.Parameters)
+	provExtCreds, err := a.dao.GetExtractedCredentials(instanceUUID.String())
```

Comment:
This reads like any APB that has been bound is *required* to have credentials stashed for both provision *and* bind. What if I have a bindable APB that only returns credentials during a bind operation? This will error out unless the provision object is some kind of empty object, which I also don't think we should be doing.